.github/workflows/go-update-commenter.yml

```diff
@@ -0,0 +1,65 @@
+name: "Go update commenter"
+
+on:
+  pull_request:
+    # Only run on PR label events (in particular not on every commit)
+    types: [ labeled ]
+
+jobs:
+  old-versions-match:
+    # Only run if the PR is labeled with 'go-update'
+    if: ${{ github.event.label.name == 'go-update' }}
+    runs-on: ubuntu-latest
+    steps:
+      # get the Go version of the target branch
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.base_ref }}
+      - name: Get former Go version
+        id: former_go_version
+        run: |
+          echo version="$(cat .go-version)" >> $GITHUB_OUTPUT
+
+      # get the Go version of the PR branch
+      - uses: actions/checkout@v3
+      - name: Get current Go version
```

Comment:
:question: Please forgive my lack of yaml/GA knowledge, but this looks like it's defining 2 steps, while I don't think it would work if it was.

Shouldn't `name` be part of the same array entry?

Reply:
This is indeed two steps and that's on purpose, the first one checkouts the repo, the second one `cat` the version file
You don't need to name each step, this is just for readability, I named the second one because it runs a script so it wouldn't have a pretty name in the Github UI

Reply:
Thanks for the explanation!